### PR TITLE
chore(babel): rename babel-plugin-import-from-lib

### PR DESCRIPTION
**Pull request by jmfrancois**
**What is the problem this PR is trying to solve?**

Context: we have two babel plugin: 
* one to be tree shaking compliant (the one renamed in this PR)
* the other one to be umd compliant

The name `@talend/babel-plugin-transform-ui-imports` is not related to it s current target: be tree shaking compliant.

**What is the chosen solution to this problem?**

Rename the plugin `@talend/babel-plugin-import-from-lib`
Make ti more generic and configurable to support lodash

**Please check if the PR fulfills these requirements**

* [ ] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: babel/plugin-import-from-lib/README.md [hunk 1]
```diff
@@ -1,23 +1,42 @@
...
 
+## Options
+
+THe plugin support the `rules` option to assign new rules with the same shape has the rules.json file:
+
+```json
+{
+    "module-name": {},
+    "module-with-lib-folder": {
+        "default": "lib"
```
Comment (frassinier) [inline babel/plugin-import-from-lib/README.md:29]: slashes are missing?

File: babel/plugin-import-from-lib/README.md [hunk 2]
```diff
@@ -1,23 +1,42 @@
...
 "plugins": [
-+    "@talend/babel-plugin-transform-ui-imports",
++    "@talend/babel-plugin-import-from-lib",
     ...
 ]
 ```
 
+## Options
+
+THe plugin support the `rules` option to assign new rules with the same shape has the rules.json file:
```
Comment (frassinier) [inline babel/plugin-import-from-lib/README.md:23]:
```suggestion
The plugin supports the `rules` option to assign new rules with the same shape in the rules.json file:
```

File: babel/plugin-import-from-lib/README.md [hunk 3]
```diff
@@ -1,23 +1,42 @@
-# @talend/plugin-transform-ui-imports
+# @talend/babel-plugin-import-from-lib
 
 This package is a babel plugin ready to be used with @talend/react-components and @talend/react-containers
 
 The object is to minimize bundle size by picking only the needed component.
```
Comment (frassinier) [inline babel/plugin-import-from-lib/README.md:5]:
```suggestion
The goal is to minimize bundle size by picking only the needed components.
```

File: babel/plugin-import-from-lib/index.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
+// https://github.com/jamiebuilds/babel-handbook/blob/master/translations/en/plugin-handbook.md
+
+const rules = require('./rules.json');
+
+function getRules() {
+	return rules;
+}
```
Comment (frassinier) [inline babel/plugin-import-from-lib/index.js:7]: what's the purpose?

File: babel/plugin-import-from-lib/index.js [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+module.exports = function transform({ types: t }) {
+	const RULES = getRules();
+	let NAMES = Object.keys(RULES);
+	function getPath(name, root) {
+		const rule = RULES[root][name];
+		const base = RULES[root].default || '/';
+		if (rule) {
+			return rule.from || `${root}${rule.path}`;
```
Comment (frassinier) [inline babel/plugin-import-from-lib/index.js:16]: We may use [path](https://nodejs.org/api/path.html#path_path_resolve_paths) utils instead or an equivalent?

File: babel/plugin-import-from-lib/index.js [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+						} else {
+							// is last so we replace
+							path.node.specifiers = [t.importDefaultSpecifier(t.identifier(spec.local.name))];
+							if (!isDefault(spec.imported.name, base)) {
+								path.node.specifiers = [t.importSpecifier(t.identifier(spec.local.name), t.identifier(spec.imported.name))];
+							}
+							path.node.source = t.stringLiteral(getPath(spec.imported.name, base));
+						}
+					} else {
+						console.warn(`WARNING: ${spec.type} are not handled. Bundle size can not be decrease`);
```
Comment (frassinier) [inline babel/plugin-import-from-lib/index.js:70]:
```suggestion
						console.warn(`WARNING: ${spec.type} are not handled. Bundle size can not be reduced`);
```

**Review by frassinier — APPROVED** [review]

File: babel/plugin-import-from-lib/index.js (hunk 5 above)
Comment (jmfrancois) [reply babel/plugin-import-from-lib/index.js:16]: it s not OS dependant i dont  want back slash on windows.

**Pull request merged**